test/e2e/windows/density.go

```diff
@@ -0,0 +1,319 @@
+/*
```

Comment:
Could you summarize what has changed (if there is any) from `e2e_node/density_test.go`?